### PR TITLE
5.9 release update

### PR DESCRIPTION
**Pull request by jsiemensLatitudegeo**
Updated VertiGIS.Mobile package to 5.9. PR build for now. Will update to final package version before completing merge.
Updated other packages to match VergiGIS.Mobile 5.9 dependencies.
Updated layout.xml files to make use of expand component.
Updated layout style attributes to use "round" instead of "map-widget".

### CODE REVIEW

File: App1/App1/layout-large.xml [hunk 1]
```diff
@@ -41,9 +46,9 @@
 
     <!-- Taskbar / layers -->
     <layer-list id="layers"/>
-
-    <!-- Taskbar / legend -->
-    <legend id="legend"/>
+    
+    <!-- Basmeap picker -->
+    <basemap-picker/>
```
Comment (ianschmitz) [inline App1/App1/layout-large.xml:51]: Intentional removal of `legend` i assume?

**Review by ianschmitz — APPROVED** [review]

**Review by Felicity-R — APPROVED** [review]

**Review by kewalker — APPROVED** [review]
Looks good - thanks for updating the layouts.

File: App1/App1/layout-large.xml [hunk 2]
```diff
@@ -41,9 +46,9 @@
 
     <!-- Taskbar / layers -->
     <layer-list id="layers"/>
-
-    <!-- Taskbar / legend -->
-    <legend id="legend"/>
+    
+    <!-- Basmeap picker -->
```
Comment (kewalker) [inline App1/App1/layout-large.xml]: Typo.

File: App1/App1/layout-large.xml (hunk 1 above)
Comment (jsiemensLatitudegeo) [reply App1/App1/layout-large.xml:51]: It's not gone, just moved into an expand on the map.

File: App1/App1/layout-large.xml (hunk 2 above)
Comment (kewalker) [reply App1/App1/layout-large.xml]: Fixed.

**Pull request merged**